### PR TITLE
add ansible uninstall and image config

### PR DESCRIPTION
**Pull request by pecameron**
Added an uninstall playbook to remove ovn
./dist/ansible/run-playbook uninstall

Added image variable in dist/ansible/hosts to select the desired image

Moved daemonset.yaml files to template files in dist/templates
Added "daemonsetyaml" target in dist/images/Makefile to expand the
templates into expected daemonset.yaml files in dist/yaml
Also, the image build also generates the daemonset.yaml files from the
template files..

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

File: dist/READMEopenshifttechpreview.md [hunk 1]
```diff
@@ -28,23 +34,34 @@ openshift_use_openshift_sdn='False'
...
 
+OVN may be removed from the cluster by running:
+```
+# ./run-playbook uninstall
+```
+
+The yaml/{ovnkube.yaml,ovnkube-master.yaml} files are now created as follows:
+```
+# cd ../images
+# make daemonsets
```
Comment (shettyg) [inline dist/READMEopenshifttechpreview.md]: I think it is better to make it something like 'make daemonsetyaml' to make it obvious.

Comment (pecameron) [reply dist/READMEopenshifttechpreview.md]: @shettyg I made your suggested changes, PTAL

Comment (pecameron) [reply dist/READMEopenshifttechpreview.md]: @shettyg @rajatchopra @dcbw more cleanups PTAL

**Comment by pecameron** [comment]
@shettyg @dcbw @squeed Is there anything further needed before merge? PTAL

**Comment by shettyg** [comment]
/lgtm for non-ansible stuff. Please do add a .gitignore file in dist/yaml directory and add the following content to it.

```
ovnkube-master.yaml
ovnkube.yaml
```

**Comment by pecameron** [comment]
@shettyg I made the change. PTAL
@dcbw @squeed PTAL

**Pull request merged**